_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -22,52 +22,29 @@ test.describe('Popular Repositories', () => {
       await expect(page.getByRole('button', { name: 'Add selected repositories' })).toBeVisible();
```

Comment:
as part of this, do you mind renaming this test from `Add popular repos` to `Test adding and removing popular repos` since it's doing more than just adding them?